monitoring/opentelemetry/src/test/java/io/quarkus/ts/opentelemetry/OpenTelemetryIT.java

```diff
@@ -58,7 +58,7 @@ public class OpenTelemetryIT {
     public void testContextPropagation() {
         int pageLimit = 10;
         String operationName = "GET /ping/pong";
-        String[] operations = new String[] { "GET /ping/pong", "GET", "GET /hello" };
+        String[] operations = new String[] { "GET /ping/pong", "GET /hello", "GET /hello" };
```

Comment:
they are there, so I don't know how array comparation works inside RestAssured, but strictly speaking this is correct. Because array with operation names contain it twice.

Reply:
See OCP CI from the FW bump or run it locally.

Reply:
In my opinion there is no need to have same endpoints. When you change `thenTraceSpanSizeMustBe` condition to match 2 endpoint it will pass. I made changes on first try and I don't know well context of that test. Please say, if I am wrong

Reply:
>  When you change thenTraceSpanSizeMustBe condition to match 2 endpoint it will pass.

This is about matching different traces with same operation name. If your theory was right that it's a same trace coming from the same request, then how would in past it be differently named? One of these must come from REST client call and we should wait for it.

Reply:
Anyway I agree that we now have hard time to detect what actually comes from the REST endpoint and what comes from REST client. Pity.

Reply:
We can use `span.kind` that must be there both from `server` and `client`. Blah, I'm bit tired, but I'll do it.

Reply:
Done.